src/flowchem/devices/knauer/knauer_autosampler.py

```diff
@@ -344,18 +334,13 @@ def aspirate(self, volume:float, flow_rate:float or int=None):
         Returns: None
 
         """
-        if not self.external_syringe_aspirate:
-            if flow_rate is not None:
-                raise NotImplementedError("Buildt in syringe does not allow to control flowrate")
-            volume = int(round(volume, 3) * 1000)
-            command_string = self._construct_communication_string(AspirateCommand, CommandModus.SET.name, volume)
-            return self._set(command_string)
-        else:
-            assert self.external_syringe_dispense is not None and self.external_syringe_ready is not None, "Make sure to set all necessary commands for external syringe"
-            self.external_syringe_aspirate(volume, flow_rate)
-            
+        if flow_rate is not None:
```

Comment:
ideally, check AS version, the one I have does not allow for it, others dp. See fowarded email